### ISSUE TITLE
go to previous codeblock in accessible view does not work

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#210592]
1. in chat, produce a response with several codeblocks
2. put your cursor in a code block that has one previous to it
3. use the left arrow action
4. 🐛 it doesn't work

<img width="908" alt="Screenshot 2024-04-17 at 11 53 59 AM" src="https://github.com/microsoft/vscode/assets/29464607/c58148b8-3849-4d10-b47b-5bc5b425ab5b">
